UI/frontend-plugins/aja-output-ui/CMakeLists.txt

```diff
@@ -1,161 +1,96 @@
 project(aja-output-ui)
 
-if(DISABLE_AJA)
-	message(STATUS "aja-output-ui plugin disabled")
-	return()
+if(NOT ENABLE_AJA)
+  return()
 endif()
 
-if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-	find_package(LibAJANTV2)
-	if (NOT LIBAJANTV2_FOUND)
-		message(STATUS "aja-output-ui plugin disabled (deps not found)")
-		return()
-	else()
-		message("aja-output-ui includes: ${LIBAJANTV2_INCLUDE_DIRS}")
-		message("aja-output-ui libs: ${LIBAJANTV2_LIBRARIES}")
-	endif()
-else()
-	message(STATUS "aja-output-ui disabled (32-bit not supported)")
-	return()
-endif()
+find_package(LibAJANTV2 REQUIRED)
 
-if(APPLE)
-	find_library(COCOA Cocoa)
-	include_directories(${COCOA})
-endif()
+add_library(aja-output-ui MODULE)
+add_library(OBS::aja-output-ui ALIAS aja-output-ui)
 
-if(UNIX AND NOT APPLE)
-	find_package(X11 REQUIRED)
-	link_libraries(${X11_LIBRARIES})
-	include_directories(${X11_INCLUDE_DIR})
+find_qt(
+  VERSION ${QT_VERSION}
+  COMPONENTS Widgets
+  COMPONENTS_LINUX X11Extras)
 
-	find_package(Qt5X11Extras REQUIRED)
+if(OS_POSIX AND NOT OS_MACOS)
+  target_link_libraries(aja-output-ui PRIVATE Qt::X11Extras)
```

Comment:
X11Extras is not needed at all for AJA. The commit adding AJA states clearly that it no longer needs it.
It seems that the CMakeLists.txt was not changed.

Commit: https://github.com/obsproject/obs-studio/commit/ce3ae8e423e87c321246ac3a08a7d8b3d3736886

Reply:
The original commit was not squashed in it.
https://github.com/obsproject/obs-studio/pull/5271/commits/e6f0e529a6d521eab6be8ae035d7c3b0560f637d